### DISCUSSION TITLE
How can u use GTS in Pokemon games

### DISCUSSION THREAD

**Discussion by yellow-flash02** [rafaelvcaetano/melonDS-android#1243] [Q&A]
I'm trying to figure out how do i use WFC connection for GTS trades in Pokemon using melonds newest version 1.9.2

**Comment by rafaelvcaetano** [comment]
You can find a guide [on this page](https://github.com/rafaelvcaetano/melonDS-android/wiki/Wi%E2%80%90Fi-Connectivity). If you don't know if you have a custom BIOS or not, then look directly at [this section](https://github.com/rafaelvcaetano/melonDS-android/wiki/Wi%E2%80%90Fi-Connectivity#setup-with-built-in-bios-and-firmware).
